core/src/test/java/com/linecorp/armeria/server/metric/PrometheusExpositionServiceTest.java

```diff
@@ -111,4 +115,30 @@ void prometheusRequests() throws InterruptedException {
         verify(logger, times(6)).isDebugEnabled();
         verify(logger, times(4)).debug(anyString(), any(), any());
     }
+
+    @Test
+    void prometheusRequestsPrometheusFormat() throws InterruptedException {
+        final WebClient client = WebClient.of(server.httpUri());
+        final HttpRequest request = HttpRequest.builder()
+            .get("/enabled")
```

Comment:
Could you fix the indentation? 😄 
You might refer to this document to set up the IDE.
https://armeria.dev/community/developer-guide#setting-up-your-ide

Reply:
hmm, importing those settings doesn't seem to be working for me in Intellij 2021.2. What's the format you are expecting? Is there a gradle command I can run to do autoformatting?

Reply:
If you've already the imported the settings, did you also try [Reformatting](https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html)?